### PR TITLE
[ISSUE #4663] Add documentation for wecom sink connector.

### LINKED ISSUE #4663: [Doc] Documentation Needed for eventmesh-connector-wecom.
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Documentation Related

Documentation Needed for eventmesh-connector-wecom. 

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct) *

### PR DESCRIPTION
**Pull request by yanrongzhen**
Fixes #4663 .

### Modifications

Add documentation for wecom sink connector.

### CODE REVIEW

File: eventmesh-connectors/eventmesh-connector-wecom/README_CN.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# eventmesh-connector-slack
+
+## SlackSinkConnector：从 eventmesh 到 slack。
+
+1. 启动你的 eventmesh-runtime。
+2. 启用 sinkConnector 并检查 `sink-config.yml`。
+3. 向 eventmesh 发送带有在 `pubSubConfig.subject` 中定义的主题消息。
+```yaml
```
Comment (Pil0tXia) [inline eventmesh-connectors/eventmesh-connector-wecom/README_CN.md:8]: The translation of the third point is not quite smooth. Can it be "使用在`pubSubConfig.subject`中指定的Topic，向EventMesh发送消息。"?

**Review by xwm1992 — APPROVED** [review]

**Review by mxsm — APPROVED** [review]

File: eventmesh-connectors/eventmesh-connector-wecom/README_CN.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# eventmesh-connector-slack
+
+## SlackSinkConnector：从 eventmesh 到 slack。
```
[the remark this answers is not in the document]
Comment (Pil0tXia) [reply eventmesh-connectors/eventmesh-connector-wecom/README_CN.md:3]: Fixed in https://github.com/apache/eventmesh/pull/4668.

File: eventmesh-connectors/eventmesh-connector-wecom/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# eventmesh-connector-wecom
+
+## WecomSinkConnector：from eventmesh to wecom。
+
+1. launch your eventmesh-runtime.
+2. enable sinkConnector and check `sink-config.yml`.
+3. send a message to eventmesh with the topic defined in `pubSubConfig.subject`
+```yaml
+pubSubConfig:
+  meshAddress: 127.0.0.1:10000
```
[the remark this answers is not in the document]
Comment (Pil0tXia) [reply eventmesh-connectors/eventmesh-connector-wecom/README.md:10]: Fixed in https://github.com/apache/eventmesh/pull/4668.

File: eventmesh-connectors/eventmesh-connector-wecom/README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# eventmesh-connector-wecom
```
[the remark this answers is not in the document]
Comment (Pil0tXia) [reply eventmesh-connectors/eventmesh-connector-wecom/README.md:1]: Fixed in https://github.com/apache/eventmesh/pull/4668.

File: eventmesh-connectors/eventmesh-connector-wecom/README_CN.md (hunk 1 above)
Comment (Pil0tXia) [reply eventmesh-connectors/eventmesh-connector-wecom/README_CN.md:8]: Fixed in https://github.com/apache/eventmesh/pull/4668.

**Pull request merged**